src/components/MetricsPane/charts/AggregateHardCoralCover.jsx

```diff
@@ -57,29 +60,34 @@ export const AggregateHardCoralCover = () => {
     })
     .filter((record) => record !== undefined)
 
-  const markerColors = hardCoralAveragesPerSurvey.map((_, index) => {
-    const colors = chartTheme.aggregateCharts.hardCoralCover.marker
+  const bins = Array.from({ length: (end - start) / binSize }, (_, i) => start + i * binSize)
 
-    if (index <= 5) {
-      return colors.low
+  const binColors = bins.map((bin) => {
```

Comment:
Continuing from the comment above, could this be `binStartValues.map((binStartValue) => …`?

Reply:
yes, I change them all, thanks